src/device/rcp/rsp/rsp_core.c

```diff
@@ -266,6 +276,7 @@ void write_rsp_regs(void* opaque, uint32_t address, uint32_t value, uint32_t mas
     {
     case SP_STATUS_REG:
         update_sp_status(sp, value & mask);
+        return;
```

Comment:
this is redundant and doesn't need to be added

Reply:
Good for clarity though.